From 2ec7eee57a47bbfd296badaa03dc20bf71b50201 Mon Sep 17 00:00:00 2001
From: Atri Bhattacharya <badshah400@gmail.com>
Date: Fri, 8 Jan 2021 01:39:00 +0100
Subject: [PATCH] configure: fix check for Poppler version >= 20.12.

Check for poppler.pc instead of poppler-splash.pc. which is no longer
installed by Poppler as of version 20.12.

https://github.com/jwilk/pdf2djvu/issues/144
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 44e6508..75852ba 100644
--- a/configure.ac
+++ b/configure.ac
@@ -73,8 +73,8 @@ do
 done
 AC_MSG_RESULT([ok])
 
-PKG_CHECK_MODULES([POPPLER], [poppler-splash >= 0.35.0])
-poppler_version=$($PKG_CONFIG --modversion poppler-splash)
+PKG_CHECK_MODULES([POPPLER], [poppler >= 0.35.0])
+poppler_version=$($PKG_CONFIG --modversion poppler)
 AC_DEFINE_UNQUOTED([POPPLER_VERSION_STRING], ["$poppler_version"], [Define to the version of Poppler])
 parse_poppler_version()
 {
